app/Http/Controllers/PlayerController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PlayerStoreRequest;
+use App\Http\Resources\PlayerCollection;
+use App\Http\Resources\PlayerResource;
+use App\Models\Player;
+use App\Services\PlayerService;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Symfony\Component\HttpFoundation\Response;
+
+class PlayerController extends BaseController
+{
+    private PlayerService $playerService;
+
+    public function __construct(PlayerService $playerService)
+    {
+        $this->playerService = $playerService;
+    }
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        return $this->sendResponse(new PlayerCollection(Player::all()), "Players retrieved successfully");
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(PlayerStoreRequest $request)
+    {
+        $validatedData = $request->validated();
+
+        $isPlayerExist = Player::where('name', $validatedData['name'])->where('second_name', $validatedData['second_name'])->exists();
```

Comment:
warto się zastanowić czy tej logiki nie przenieść do serwisu i np skorzystać z abort_if(), które zwróci Ci HttpException, lub napisać własne i wyłapać je w kontrolerze

Reply:
na ten moment ją za komentuje, bo nie ma większego sensu bo jeśli zmieniasz tylko ligę to wywali też error przez to, że imię i nazwisko zawodnika jest takie same, na ten moment w ligach nie było nikogo o tych samych imionach i nazwiskach wiec raczej z tymi nie ma problemu, najwyżej doda się kolumnę z drugim imieniem lub ksywką, żeby rozrużniać